oauth2_provider/constants.py

```diff
@@ -41,6 +41,10 @@
 SCOPE_NAME_DICT = dict([(name, value) for (value, name) in SCOPES])
 SCOPE_VALUE_DICT = dict([(value, name) for (value, name) in SCOPES])
 
+# since this package explicitly supports the 'id_token' response type, override
+# the base package's default.
+DEFAULT_RESPONSE_TYPE_CHOICES = ('code', 'token', 'id_token')
+RESPONSE_TYPE_CHOICES = getattr(settings, 'OAUTH_RESPONSE_TYPE_CHOICES', DEFAULT_RESPONSE_TYPE_CHOICES)
```

Comment:
well, after a second look i'm inclined to leave this as is.  appending 'id_token' will not have any different effect on the value in this module than explicitly defining the tuple value we want.  import issues in edx-platform can be fixed there.  again, whatever the host django app defines for `OAUTH_RESPONSE_TYPE_CHOICES` should take precedence anyway.